llarp/rpc/rpc_server.cpp

```diff
@@ -94,6 +94,210 @@ namespace llarp::rpc
     }
   }
 
+  void
+  HandleJSONCommand(
+      oxenmq::Message& msg,
+      std::string replyCmd,
+      std::function<void(nlohmann::json, ReplyFunction_t)> handleRequest)
+  {
+    const auto maybe = MaybeParseJSON(msg);
+    if (not maybe.has_value())
+    {
+      msg.send_back(replyCmd, CreateJSONError("failed to parse json"));
```

Comment:
very minor style thought: "JSONError" looks like it could be a type and this could just be a constructor, is the "Create" superfluous?